engine/execution/state/delta/delta.go

```diff
@@ -89,7 +89,7 @@ func (d Delta) MarshalJSON() ([]byte, error) {
 	return json.Marshal(m)
 }
 
-func (d Delta) UnmarshalJSON(data []byte) error {
+func (d *Delta) UnmarshalJSON(data []byte) error {
```

Comment:
Why changing this?

Reply:
as the [commit message indicates](https://github.com/onflow/flow-go/pull/990/commits/2767eadab1833dad69ac1e67a3015b7ab080d0fa), the assigment at line 106 occurs on an immediately-lost copy of the Delta otherwise.